src/patterns/components/dividers/angular/info/base.hbs

```diff
@@ -0,0 +1,21 @@
+<p class="sprk-u-mbm">See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+  <thead>
+  <tr>
+    <th>Input</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td class="sprk-u-FontWeight--bold">additionalClasses</td>
+    <td>string</td>
+    <td>Allows a string of classes to add, in addition to the spark classes. Intended for overrides.</td>
```

Comment:
We should go through and make sure we are capitalizing Spark when we are done with docs.